include/dxc/DxilContainer/RDAT_SubobjectTypes.inl

```diff
@@ -9,6 +9,13 @@
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
+// clang-format off
```

Comment:
Same here. Clang formatting this off in the branch makes this harder to review.

Reply:
Pull request to turn off clang formatting in main for this file: https://github.com/microsoft/DirectXShaderCompiler/pull/5986

Reply:
If you check "hide whitespace", this file becomes easy to review.  All that's changed is this at the top, indentation, and local reformatting, which reflowed some lines below.

Reply:
Pull request to turn off clang formatting is completed in main and this PR was updated with a merge from main that contains this commit.